Tests/conf.json

```diff
@@ -3409,4 +3403,4 @@
             }
         }
     }
-}
+}
```

Comment:
Could you please remove the changes (even if they are just blank lines) from this file - conf.json?
It causes conflicts.